webui/react/bin/try-for-flakes.sh

```diff
@@ -0,0 +1,24 @@
+# eg ./bin/try-for-flakes.sh true "npm run test -- --watchAll=false src/components/Timeago.test.tsx"
+
+# if tests are re-runnable
+re_runnable=${1:-false}
```

Comment:
we probably don't need to differentiate between rerunnable and not anymore but this works as is. We'll know what we need from this next time we're looking for a flake

Reply:
something like this look ok?
```
# eg ./bin/try-for-flakes.sh "npm run test -- --watchAll=false src/components/Timeago.test.tsx"

test_cmd=${1:-"npm run test -- --watchAll=false"}

c=0

while true; do
  c=$((c+1))
  echo "run #$c"
  ${test_cmd}
done

echo "result: test failure at run #$c of $(git rev-parse --short HEAD)"
# TODO use trap to show successful result upon SIGTERM SIGINT
```